internal/integration/integration.go

```diff
@@ -31,8 +31,31 @@ import (
 	"github.com/google/exposure-notifications-server/internal/storage"
 	"github.com/google/exposure-notifications-server/pkg/keys"
 	"github.com/google/exposure-notifications-server/pkg/secrets"
+
+	authorizedappmodel "github.com/google/exposure-notifications-server/internal/authorizedapp/model"
+	exportdatabase "github.com/google/exposure-notifications-server/internal/export/database"
+	exportmodel "github.com/google/exposure-notifications-server/internal/export/model"
+)
+
+const (
+	exportDir = "my-bucket"
 )
 
+// NewTestServer sets up clients used for integration tests
+func NewTestServer(tb testing.TB, ctx context.Context, exportPeriod time.Duration) (*serverenv.ServerEnv, *EnServerClient, *database.DB) {
+	env, client := testServer(tb)
```

Comment:
Does this need to be `TestServer`?

Reply:
Never mind.  Think I got something wrong when I checked out and merged your branch ;)

Reply:
:)